sdk/cosmos/benchmark/pom.xml

```diff
@@ -31,158 +10,171 @@
     <artifactId>azure-cosmos-parent</artifactId>
     <version>3.0.0</version>
   </parent>
-  
+
+  <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-cosmos-benchmark</artifactId>
-  <name>Async SDK for SQL API of Azure Cosmos DB Service - Benchmarking tool</name>
-  <description>Benchmarking tool for Async SDK for SQL API of Azure Cosmos DB Service</description>
 
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-  </properties>
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>exec-maven-plugin</artifactId>
-        <version>1.2.1</version>
-        <configuration>
-          <mainClass>com.azure.data.cosmos.benchmark.Main</mainClass>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <version>2.2</version>
-        <configuration>
-          <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-          </descriptorRefs>
-          <archive>
-            <manifest>
-              <mainClass>com.azure.data.cosmos.benchmark.Main</mainClass>
```

Comment:
we need the executable jar with dependencies for benchmark tool.
why did we remove this? Is this moved somewhere else?

Reply:
Agreed - this should be added back.

Reply:
Some plugins and dependencies were commented out. 
I've uncommented this one in both benchmark and examples project poms.